google/cloud/bigquery/_pandas_helpers.py

```diff
@@ -18,8 +18,8 @@
 import functools
 import logging
 import warnings
-
```

Comment:
This empty line divides standard library imports from third party imports, and should stay.